packages/react-native/third-party-podspecs/fmt.podspec

```diff
@@ -8,14 +8,14 @@ fmt_git_url = fmt_config[:git]
 
 Pod::Spec.new do |spec|
   spec.name = "fmt"
-  spec.version = "9.1.0"
+  spec.version = "11.0.2"
   spec.license = { :type => "MIT" }
   spec.homepage = "https://github.com/fmtlib/fmt"
   spec.summary = "{fmt} is an open-source formatting library for C++. It can be used as a safe and fast alternative to (s)printf and iostreams."
   spec.authors = "The fmt contributors"
   spec.source = {
     :git => fmt_git_url,
-    :tag => "9.1.0"
```

Comment:
@cipolleschi Are we bumping FMT also? If so this needs to be mentioned in changelog

Reply:
yes, it is required by folly.